src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import Login from './components/Login.js';
+import SignUp from './components/SignUp.js';
+
+function App() {
+  return (
+    <div className="App">
+      <Route 
+        exact
+        path="/login"
+        render={props => <Login {...props} /> }
+      />
+      <Route
+        exact
+        path="/signup"
+        render={props => <SignUp {...props} /> }
+      />
```

Comment:
Worth noting that you can just use `component={Login}` and `component={SignUp}` here instead of using render (since you aren't passing down any custom props.